hack/windows/Prepare-Node.ps1

```diff
@@ -50,6 +56,20 @@ If (Get-Service kubelet -ErrorAction SilentlyContinue) {
     exit 0
 }
 
+if ($ContainerRuntime -eq "Docker") {
```

Comment:
Do we support default value if the user don't add ContainerRuntime parameter?

Reply:
Yes , check  line 40
[parameter(HelpMessage="Container runtime that Kubernets will use")]
    [ValidateSet("ContainerD", "Docker")]
    [string] $ContainerRuntime = "Docker". ---- this specifies default runtime